itests/kit/mir_tools.go

```diff
@@ -74,7 +73,7 @@ func CheckNodesInSync(ctx context.Context, from abi.ChainEpoch, baseNode *TestFu
 
 // waitNodeInSync waits when the tipset at height will be equal to targetTipSet value.
 func waitNodeInSync(ctx context.Context, height abi.ChainEpoch, targetTipSet *types.TipSet, node *TestFullNode) error {
-	timeout := time.After(5 * time.Second)
+	timeout := time.After(10 * time.Second)
```

Comment:
Should we adjust the timeout dynamically?

Reply:
Fixed